### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Spica8080**
Check out my Home Page additions!

### CODE REVIEW

**Comment by AndresTIY** [comment]
looks good, there's nothing here I don't disagree with. once we get this merged, I will change the class names to match!

File: index.html [hunk 1]
```diff
@@ -5,6 +5,16 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
     <link rel="stylesheet" href="main.css">
     <title>Battle Game Project</title>
+<<<<<<< HEAD
+  </head>
```
Comment (joebumbulis) [inline index.html]: Be sure that all your html for the site is below the head.

File: index.html [hunk 2]
```diff
@@ -5,6 +5,16 @@
...
+  <body>
+    <div class="start-game">
+      <input class="player-name-input" type="text" name="player-name">
+      <button class="begin-journey-button"type="button" name="start-game-button">Begin Journey</button>
+      <div class="background-img-homepage">
+        <img src="images/richard-tilney-bassett-183124.jpg">
```
Comment (joebumbulis) [inline index.html]: I think background images should go in the CSS since it's just styling and not functional for the site.

**Review by joebumbulis — COMMENTED** [review]
There's a couple things that need to be changed before merge, see my notes.

File: index.html (hunk 1 above)
Comment (Spica8080) [reply index.html]: Thanks Joe I had under body in my branch! Updated now 👍

File: index.html [hunk 3]
```diff
@@ -5,13 +5,28 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
     <link rel="stylesheet" href="main.css">
     <title>Battle Game Project</title>
+<<<<<<< HEAD
+  </head>
```
Comment (joebumbulis) [inline index.html]: It looks like the head is empty now. Which is good that your body html is out, but the head needs to have the CSS, title, charset, etc.

**Comment by bekahwhittle** [comment]
looks good now!

**Pull request merged**